README.md

```diff
@@ -19,6 +19,18 @@ All modules in this repository adhere to the module structure defined in the the
 
 See deployment steps in the [Deployment Guide](DEPLOYMENT.md).
 
+## Project Manifests
+
+End-to-end example use-cases built using modules in this repository.
+
+| Type                                                                                                  | Description                                                                                                                                                                                                  |
+|-------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [MLOps with Amazon SageMaker](manifests/sagemaker/mlops-sagemaker/)                                   | Set us environment for MLOps with Amazon SageMaker. Deploy secure Amazon SageMaker Studio Domain, and provisions SageMaker Project Templates using Service Catalog, including model training and deployment. |
```

Comment:
Nit: Should this read "Set up environment" instead of "Set us environment"?

Reply:
Corrected, thanks!